.github/workflows/build_release.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and publish release on new tag
+on:
```

Comment:
Should we limit to main branch 🤔

Reply:
It only builds on tags, so there's technically no branch.
Only people who are allowed to write/do releases should have permission to push tags